Lib/http/server.py

```diff
@@ -1256,39 +1253,48 @@ def test(HandlerClass=BaseHTTPRequestHandler,
 
 if __name__ == '__main__':
     import argparse
+    import contextlib
 
     parser = argparse.ArgumentParser()
     parser.add_argument('--cgi', action='store_true',
-                       help='Run as CGI Server')
+                        help='Run as CGI Server')
     parser.add_argument('--bind', '-b', metavar='ADDRESS',
                         help='Specify alternate bind address '
                              '[default: all interfaces]')
     parser.add_argument('--directory', '-d', default=os.getcwd(),
                         help='Specify alternative directory '
-                        '[default:current directory]')
+                        '[default: current directory]')
+    parser.add_argument('--protocol', '-p', default='HTTP/1.0',
+                        help='Specify highest HTTP version supported '
+                        'by the server [default: HTTP/1.0]')
     parser.add_argument('port', action='store',
                         default=8000, type=int,
                         nargs='?',
                         help='Specify alternate port [default: 8000]')
     args = parser.parse_args()
     if args.cgi:
-        handler_class = CGIHTTPRequestHandler
+        HandlerClass = CGIHTTPRequestHandler
```

Comment:
The change from `handler_class` to `HandlerClass` feels like an unnecessary stylistic change and also wrong. I'd expect `HandlerClass` to be used when it's the actual class name but `handler_class` to be used when it's a reference to another class.

Reply:
I suppose this was done to pass later as `test(HandlerClass=HandlerClass, ...)`, matching naming for ServerClass.

Reply:
@jaraco Yes that was done to follow the case used in `http.server.test(HandlerClass=…, ...)` and `socketserver.BaseServer(RequestHandlerClass=…, …)` which also define identifiers.